### ISSUE TITLE
Crash: printf with width equal to INT_MIN

### ISSUE THREAD

**Issue by ch3root** [TrustInSoft/tis-interpreter#73]
Source code:
```c
#include <limits.h>
#include <stdio.h>

int main()
{
  printf("%*d\n", INT_MIN, 1);
}
```

tis-interpreter (274be6de79973f11ab62a3c77d0981882734cb2d) output:
```
[value] Analyzing a complete application starting at main
[value] Computing initial state
[value] Initial state computed
Segmentation fault
```

**Comment by ch3root** [comment]
Well, I haven't yet updated my OCaml so this is probably the same issue as #43.

**Comment by ch3root** [comment]
Yes, this is fixed by patching frama-c as in #43. Sorry for the noise.
